src/repositories/schemas.ts

```diff
@@ -25,3 +25,8 @@ export interface VotingOption {
   identifier: string;
   label: string;
 }
+
```

Comment:
I would suggest to split it in

CreationAdmin
and 
Admin extends CreationAdmin

to add _id to the interface

Reply:
why exactly do you suggest splitting it into two seperate interfaces

Reply:
It would be a good idea to also return the _id in the json response and if we want to use the same interface for that it should also be a parameter _id in the interface but while creating a admin you should not be able to add a _id

Reply:
I just put the Interface for creation into the controller itself, similar to the implementation of other controllers. 
But might make sense to restructure that sometime in the future